tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -362,14 +377,14 @@ def test_load_manifest(self):
             isinstance(helm_template, HelmTemplate) for helm_template in helm_templates
         )
         assert helm_templates[0].filepath == "chart/templates/test3a.yaml"
-        assert helm_templates[0].template == {"data": [{"a": 1}, {"b": 2}]}
         assert helm_templates[1].filepath == "chart/templates/test3b.yaml"
         assert helm_templates[1].template == {"foo": "bar"}
 
     def test_get_manifest(self, run_command: MagicMock, mock_get_version: MagicMock):
         helm_wrapper = Helm(helm_config=HelmConfig())
         run_command.return_value = """Release "test-release" has been upgraded. Happy Helming!
 NAME: test-release
+MANIFEST:
```

Comment:
why did you add this line, is it guaranteed that the Helm template always includes this line?
otherwise, I'd say we should leave the test as-is.

Reply:
I added this because this message represents the stdout of `helm upgrade --install,` and this command outputs these headers, HOOKS, MANIFEST, and NOTES (optional).
On the other hand, the command `helm get manifest` only prints the manifests without any header. That is why I added the check if the `MANIFEST:` header exists, otherwise the start should be `---`

Reply:
Actually this is the output of `helm get manifest` I'll correct the string :D

Reply:
I would say revert this line or even remove the whole header if that's the output format of `helm get manifest`